examples/terraform/aws-private/output.tf

```diff
@@ -47,32 +52,95 @@ output "kubeone_workers" {
   value = {
     # following outputs will be parsed by kubeone and automatically merged into
     # corresponding (by name) worker definition
-    "${var.cluster_name}-pool1" = {
+    "${var.cluster_name}-${local.zoneA}" = {
```

Comment:
Should we create a node in each AZ for non-private AWS setup too? There we create only one MachineDeployment, with one replica, in the first AZ.

Reply:
yeah, spreading control plane nodes across AZs but having workers in only 1 AZ is kinda strange. But this will require changes in e2e tests (the place where we count ready nodes).

Reply:
> But this will require changes in e2e tests (the place where we count ready nodes).

That should be easy, you just have to increase numbers here:
* https://github.com/kubermatic/kubeone/blob/master/test/e2e/upgrade_test.go#L62
* https://github.com/kubermatic/kubeone/blob/master/test/e2e/conformance_test.go#L50

Do you want to do that now or in a follow-up?

Reply:
someday ©